centaur/src/main/resources/standardTestCases/drs_tests/drs_usa_jdr.inputs

```diff
@@ -20,8 +20,14 @@
     # Meanwhile the gsUri for file2 may be passed directly to specialized command line programs that can read the file
     # name and extension from the end of the GCS path.
     #
+    # gcsFile1 & 2 are non-DRS URLs to the same files. The DRS localizer treats GCS urls differently than those
+    # obtained from a DRS resolution, so these are used to verify that GCS functionality works alongside DRS functionality.
```

Comment:
Is the idea here to make sure that we're only calling the DRS Localizer with actual DRS files?

Reply:
This was just a slightly easier implementation: These files were already in a public bucket centaur could access, so I re-used them. It shouldn't matter that they're the same files that the DRS url resolves to. 

This comment is confusing, though. I'll remove the `to the same files` part of the comment since it's distracting from the intention of the test.

Reply:
Janet and I chatted offline about this. Convinced that we're getting enough coverage already with existing unit + centaur tests and this addition isn't necessary.